### ISSUE TITLE
Any plan to include more information in `Symbol`

### ISSUE THREAD

**Issue by metaworm** [getsentry/symbolic#635]
Currently, the symbol object contains only its name and range.

Sometimes we want to know more information about a symbol, such as type id, symbol flags.

Symbol flags indicates whether the symbol is from export table or import table, whether the symbol is a function or the symbol is a global variable, etc.

Is there has any plan or approach to support these features?

**Comment by Swatinem** [comment]
I don’t think we have plans to support this. A question would also be if these properties exist and make sense for all the different formats we support.
